metadata-models/src/main/pegasus/com/linkedin/ermodelrelation/ERModelRelationProperties.pdl

```diff
@@ -0,0 +1,79 @@
+namespace com.linkedin.ermodelrelation
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.CustomProperties
+
+/**
+ * Properties associated with a ERModelRelation
+ */
+@Aspect = {
+  "name": "eRModelRelationProperties"
+}
+record ERModelRelationProperties includes CustomProperties {
+
+  /**
+   * Name of the ERModelRelation
+   */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  name: string
+
+  /**
+  * First dataset in the ermodelrelation (no directionality)
+  */
+   @Relationship = {
+     "name": "ermodelrelationA",
+     "entityTypes": [ "dataset" ]
+   }
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  datasetA: DatasetUrn
+
+  /**
+  * Second dataset in the ermodelrelation (no directionality)
+  */
+   @Relationship = {
+     "name": "ermodelrelationB",
+     "entityTypes": [ "dataset" ]
+   }
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  datasetB: DatasetUrn
+
+  /**
+   * ERModelRelationFieldMapping (in future we can make it an array)
+   */
+  ermodelrelationFieldMapping: ERModelRelationFieldMapping
```

Comment:
can we rename and update this field to look like this?:
```
relationshipFieldMappings: array[RelationshipFieldMapping]
```

and then create a new file called `RelationshipFieldMapping` that looks like this:
```
/**
 * Individual Field Mapping of a relationship- one of several
 */
record RelationshipFieldMapping {
  /**
   * All fields from dataset A that are required for the join, maps to bFields 1:1
   */
  sourceField: SchemaFieldPath

  /**
   * All fields from dataset B that are required for the join, maps to aFields 1:1
   */
  destinationField: SchemaFieldPath
}
```

then we can get rid of `ERModelRelationFieldMapping.pdl` and `FieldMap.pdl`

Reply:
Did not change this.

Reply:
can you explain a little more why not? here, I'm just trying to be consistent across the existing models we have at acryl and what's defined here